src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +53,29 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
-        System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+
+        User user = new User();
+        user.setEmail("nikita@gmail.com");
+        user.setSalt(HashUtil.getSalt());
+        user.setPassword(HashUtil.hashPassword("1234Qwerty", user.getSalt()));
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        userService.add(user);
```

Comment:
You have AuthentificationService to register the user
Within the UserService, the user password is hashed during the execution of the add() method

Reply:
good point, fix it